internal/storage/v2/api/tracestore/reader.go

```diff
@@ -88,6 +88,29 @@ type TraceQueryParams struct {
 	NumTraces     int
 }
 
+// FindTraceIDsChunk represents a chunk of trace IDs that match specific query parameters.
+type FindTraceIDsChunk struct {
+	// TraceIDs is a slice of trace IDs that match the query parameters.
+	TraceIDs []pcommon.TraceID
```

Comment:
this should be symmetrical with GetTraceParams, where timestamps are attached to each trace ID